proptest/Cargo.toml

```diff
@@ -63,62 +64,15 @@ break-dead-code = []
 [dependencies]
 bitflags = "1.0.1"
 
-# [dependencies.hashmap_core]
-# version = "0.1.5"
-# optional = true
-
-[dependencies.lazy_static]
```

Comment:
Tracking issue for the removed dependencies?

Reply:
#1640